addCourse.html

```diff
@@ -0,0 +1,17 @@
+<html>
+<form>
+    <fieldset>
+        <legend>Add Course</legend>
+        Title: <input type="text" name="Title" size="50" maxlength="101" required><br>
+        Date: <input type="date" name="Date" size="11" maxlength="11" required><br>
+
+        Description: <input type name="Description" size="50" maxlength="300" required><br>
+        NameOfTrainer: <input type="text" name="NameOfTrainer" size="50" maxlength="100" required><br>
```

Comment:
The customer changed the acceptance criteria very recently so name of trainer should no longer be present in the frontend.